### DISCUSSION TITLE
QB in Chromium version of 87.0.4280.144

### DISCUSSION THREAD

**Discussion by vollowx** [qutebrowser/qutebrowser#7826] [Q&A]
This is really such a low version of Chromium that many features are not usable including CSS `currentColor`.

I'm currently using NixOS, so I am not sure if this is normal. If it is, any resolutions?

**Comment by The-Compiler** [comment]
> This is really such a low version of Chromium that many features are not usable including CSS `currentColor`.

That [seems to have been introduced](https://caniuse.com/currentcolor) in Chromium 4?

> I'm currently using NixOS, so I am not sure if this is normal. If it is, any resolutions?

It's because of the migration from Qt 5 to Qt 6, which was a massive undertaking - writing this from a Chromium 108-based qutebrowser. We're [almost finished](https://github.com/qutebrowser/qutebrowser/milestones/v3.0.0) at this point, but you might want to run from the git main until v3.0.0 is out.

**Reply by vollowx** [reply]
Thanks, I have actually forgotten what is missing is the native `color-scheme` support. Anyway, I will wait for the new version, for this is really a great browser :)

**Reply by The-Compiler** [reply]
That [seems to have been](https://caniuse.com/mdn-css_properties_color-scheme) introduced in 81, so probably not the culprit either, FWIW :wink:

And yeah, I do realize this is way overdue. Usually qutebrowser is ready when a new Qt version is released (see e.g. #7745), but the Qt 6 support was quite a beast, with initial work starting over 2.5 years ago :sweat_smile: (#5395, #7202). Thanks for the nice words! :)
